sdk-c/Cargo.toml

```diff
@@ -16,10 +16,10 @@ crate-type = ["staticlib"]
 bincode = "1.2.1"
 bs58 = "0.3.1"
 libc = "0.2.69"
-rand_chacha = "0.1.1"
-rand_core = { version = ">=0.2, <0.4", default-features = false }
```

Comment:
@jackcmay does it look ok? I'm not sure why it's a range of versions before.

Reply:
@sakridge Good question, I'm not familiar with why that was done, @jstarry might know

Reply:
I think this is a question for @TristanDebrunner , actually. I may be talking to him tomorrow, so I'll see if he remembers. That said, I don't think we have any downstream consumers of sdk-c, so I wouldn't be too worried about updating it.